site/docs/blog/posts/2023-05-namespace-enforcement.md

```diff
@@ -1,6 +1,14 @@
+---
+date: 2023-05-08
+authors:
+  - abhat
+---
+
 # Namespace enforcement in Nessie
 
 Starting from Nessie version 0.52.3, it is required to have existing Namespaces before creating or committing to tables.  
+<!-- more -->
```

Comment:
What does this mean / do?

Reply:
https://squidfunk.github.io/mkdocs-material/setup/setting-up-a-blog/#adding-an-excerpt